.gitignore

```diff
@@ -2,3 +2,4 @@
 build
 composer.phar
 composer.lock
+.idea/
```

Comment:
Please remove this as it should be in your global ignore file.

Reply:
Sure, removing it now!